sgv2-service-common/src/main/java/io/stargate/sgv2/common/cql/builder/Value.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright DataStax, Inc. and/or The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.sgv2.common.cql.builder;
+
+public interface Value<T> {
```

Comment:
A small Javadoc might make sense here... also, name is very generic. I don't have a feel of if it'd be exposed -- probably not as it's just a base/tag interface? -- and if not, fine as-is I suppose. Maybe `CqlValue` would be better name regardless?